content/docs/intro/concepts/programming-model.md

```diff
@@ -802,7 +930,7 @@ Secrets are not yet available in Go. See <https://github.com/pulumi/pulumi/issue
 
```

Comment:
Do we want to add a `<div class="language-prologue-csharp"></div>` here with a paragraph or couple of bullets on how to programmatically create secret values in C#?

If so, the `{{< langchoose >}}` on line 910 needs to be changed to `{{< langchoose csharp >}}` to make the C# tab visible.

Reply:
Will update.